Sources/CustomerCenter/CustomerCenterConfigData.swift

```diff
@@ -15,6 +15,9 @@
 
 import Foundation
 
+// swiftlint:disable missing_docs
+public typealias RCColor = PaywallColor
```

Comment:
Hmm I wonder if we should make this a more global typealias (outside the `CustomerCenter` folder)... Even better, we might want to have the main type be `RCColor` and change everything to that and just keep `PaywallColor` as a typealias instead? But that can come on future PRs as well.